iocage/lib/ioc_start.py

```diff
@@ -355,10 +352,7 @@ def __start_jail__(self):
         os_path = f"{self.path}/root/dev/log"
 
         if not os.path.isfile(os_path) and not os.path.islink(os_path):
-            original_path = os.getcwd()
-            os.chdir(f"{self.path}/root/dev")
-            os.symlink("../var/run/log", "log")
-            os.chdir(original_path)
+            os.symlink("../var/run/log", os_path)
```

Comment:
I don't think this will work a s the symlink will be invalid from the jail side. As it is linked to the path from the hosts point of view.

Reply:
os.symlink allows to specify relative paths while creating a link on an absolute path:

```python3
>>> import os
>>> os.makedirs("/tmp/one/two")
>>> os.makedirs("/tmp/three/four")
>>> os.symlink("../../three", "/tmp/one/two/five")
>>> os.listdir("/tmp/one/two")
['five']
>>> os.listdir("/tmp/one/two/five")
['four']
```